artemis-server/src/main/java/org/apache/activemq/artemis/spi/core/protocol/AbstractProtocolManager.java

```diff
@@ -50,13 +52,15 @@ protected String invokeInterceptors(final List<I> interceptors, final P message,
 
    @Override
    public void setAnycastPrefix(String anycastPrefix) {
+      ActiveMQServerLogger.LOGGER.info("Setting a prefix", new Exception());
```

Comment:
Remove this please.. it looks like Debug you added for your own benefit and it was forgotten here.

I actually got a big exception on the broker I jsut started with your PR checked out, and I thought it had an error.